src/pages/Settings/TwoStepVerificationPage.vue

```diff
@@ -205,8 +201,57 @@ export default {
 			}
 			return 'You\'ll be asked for a verification code when accessing you Kiva account.';
 		}
+
 	},
 	methods: {
+		gatherMfaEnrollments() {
+			this.kvAuth0.checkSession()
+				.then(() => this.kvAuth0.getMfaManagementToken())
+				.then(token => {
+					return this.apollo.query({
```

Comment:
In my testing this method was correctly only called after `removeMfaMethod` had completed, however instead of making a new request to the network it just read the values from the cache, which is probably why it seemed like it was being called before `removeMfaMethod` had completed. To fix that, you can add `fetchPolicy: 'network-only',` to the options for this query.